products/sce/gateway/lb-options.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: 'Load Balancing Options'
+---
+
+When you enable the container gateway during container group creation, several load balancing (LB) options determine how
+requests are forwarded to the container instances, including algorithms, concurrency and timeouts. These settings cannot
+be changed once the container group is created. To modify the container gateway settings, you can replicate the existing
```

Comment:
I think the appropriate wording here would be "duplicate" rather than "replicate," just to follow along with the label our button has in the portal.

Reply:
Changed, two places